helm-chart/renku-graph/charts/jena/templates/lucene-config.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
     chart: {{ template "jena.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
-type: Opaque
 data:
```

Comment:
I guess it should be a list?

Reply:
no, what I wanted to say is that config-maps don't have types at all.